src/storage/commands.rs

```diff
@@ -12,23 +12,25 @@ use crate::storage::types::{Key, Mutation};
 pub const CMD_TAG_GC: &str = "gc";
 pub const CMD_TAG_UNSAFE_DESTROY_RANGE: &str = "unsafe_destroy_range";
 
-pub struct PointGetCommand {
+/// A 'get' command which is part of a batched get operation.
+pub struct BatchedGetCommand {
```

Comment:
I think this renaming doesn't seems very important, but I'm afraid it may be misunderstood as a batch of many get commands.

Reply:
I hope it's clear from the struct layout or docs that is not the case. I found `PointGetCommand` to be a very bad name since even from looking at the declaration it is not clear what it is.

Reply:
I think `PointGetCommand` is better... It's just a command like other commands in `src/storage/commands.rs`. It doesn't know how the user preprocesses it.

Reply:
But what does "Point" mean? And afaict it is only used in batches, which is why it is not part of the Command enum. Maybe the name is fine but I don't understand how it is being used. In which case we can document it better.